world/content/riverport/central.ev

```diff
@@ -0,0 +1,353 @@
+#
+# Central area of Riverport
+#
+# start in "Limbo"
+#
+@tel Limbo
+
+######
+# create the first room
+#
+@dig Kai River Bridge;riverport#01 = riverport
+#
+# describe the entrance into the game from limbo
+#
+@desc riverport =
+ This is the entrance to the |mAinneve|n game world.
+#
+# move to the room
+#
+riverport
+#
+# set zone tag
+#
+@tag riverport#01 = riverport : zone
+#
+# set terrain
+#
+@terrain = EASY
+#
+# set range field
+#
+@rangefield = (4, 3)
+#
+# next, set its description
+#
+@desc
+
+ You stand in the middle of an ancient |wbridge|n spanning the |gKai River|n.
+
+ The |wriver|n runs from the north to the south, and a well-traveled |wroad|n
+ runs east and west.
+
```

Comment:
I agree with staying consistent, and looking at this line above, there may be some instances of `|r` and some of `|g`, so that should be addressed.

On the [Builder Docs](https://github.com/evennia/ainneve/wiki/Builder-Documentation#colors) wiki page, I did start documenting the color choices that have made it into master so far. 

I do like the idea of a more markup-based color-theming system, but that seems like a more systemic change than this builder task calls for, and should perhaps be addressed as a separate issue/feature request.